Assets/_BForBoss/_Core/Scripts/Environment/PlayerTriggerAreaEffect.cs

```diff
@@ -0,0 +1,127 @@
+using Perigon.Character;
+using Perigon.Entities;
+using Perigon.Utility;
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace BForBoss
+{
+    [RequireComponent(typeof(Collider))]
+    public class PlayerTriggerAreaEffect : MonoBehaviour
+    {
+        private enum PlayerEffect
+        {
+            Damage,
+            Heal
+        }
+
+        private enum PlayerModifier
+        {
+            SlowDown,
+            SpeedUp
+        }
+
+        [Title("Player Effect", "Effect on Player over time", TitleAlignments.Split)]
+        [SerializeField] private PlayerEffect _playerEffect = PlayerEffect.Damage;
+        [SerializeField, Min(0), Tooltip("The amount to either damage or heal the player")] private int _effectAmount;
+        [SerializeField, Min(0.0f), Tooltip("Duration between player getting effected on")] private float _secondsBetweenEffect = 1.0f;
+
+        [Title("Player Modifier", "Modification on Player's stats", TitleAlignments.Split)]
+        [SerializeField] private PlayerModifier _playerModifier = PlayerModifier.SlowDown;
+        [SerializeField, Range(0, 100), Tooltip("Modification on Player stats as a percentage")]
+        private int _modificationAmount = 50;  
```

Comment:
It's not really clear what stats this is effecting. Is health? or speed? 
Also percentages are normally between 0 - 1

Reply:
Can definitely change it to 0 - 1